exercises/harry-potter/description.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
You will not find it in https://github.com/rpottsoh/x-common/tree/rpottsoh-patch-2/exercises/book-store .

You will find it in https://github.com/rpottsoh/x-common/tree/rpottsoh-patch-2/exercises/harry-potter .